.gitignore

```diff
@@ -32,4 +32,19 @@ Makefile
 *.[do]
 *.l[doa]
 *.loT
-*.orig
\ No newline at end of file
+*.orig
+*.mexmaci64
+src/matlab/vigraBoundaryTensor.m
```

Comment:
are these types of m files automatically generated?

Reply:
Exactly, auto generated by Matlab . I added them to .gitignore to prevent checkin by accident